modules/es-extensions/publisher/extensions/app/greg_swagger/pages/index.jag

```diff
@@ -0,0 +1,193 @@
+          <link href='../extensions/app/greg_swagger/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
+          <link href='../extensions/app/greg_swagger/css/reset.css' media='print' rel='stylesheet' type='text/css'/>
+          <link href='../extensions/app/greg_swagger/css/print.css' media='print' rel='stylesheet' type='text/css'/>
+          <script src='../extensions/app/greg_swagger/lib/jquery-1.8.0.min.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/jquery.slideto.min.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/jquery.wiggle.min.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/handlebars-2.0.0.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/underscore-min.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/backbone-min.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/js/swagger-ui.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/highlight.7.3.pack.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/marked.js' type='text/javascript'></script>
+          <script src='../extensions/app/greg_swagger/lib/swagger-oauth.js' type='text/javascript'></script>
+
+          <script type="text/javascript">
+            $(function () {
+      var url = window.location.search.match(/url=([^&]+)/);
+      if (url && url.length > 1) {
+        url = decodeURIComponent(url[1]);
+      } else {
+        url = "<%=
+    url
+    %>";
+         }
+
+         window.swaggerUi = new SwaggerUi({
+           swaggerContent: <%=
+    jsonValue
+    %>,
+           swaggerChildren: <%=
+    childAPIs
+    %>,
+               url: url,
+               dom_id: "swagger-ui-container",
+               supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
+               onComplete: function(swaggerApi, swaggerUi){
+                 if(typeof initOAuth == "function") {
+                   initOAuth({
+                     clientId: "your-client-id",
+                     realm: "your-realms",
+                     appName: "your-app-name"
+                   });
+                 }
+
+                 $('pre code').each(function(i, e) {
+                   hljs.highlightBlock(e)
+                 });
+
+                 addApiKeyAuthorization();
+               },
+               onFailure: function(data) {
+                 log("Unable to Load SwaggerUI");
+               },
+               docExpansion: "none",
+               apisSorter: "alpha",
+               showRequestHeaders: false
+             });
+
+             function addApiKeyAuthorization(){
```

Comment:
Thanks @splinter for pointing out this formatting issue, I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/bfa37074378e690eb2cb40ecf3d5de50e684f9e3)